site/pages/api/delete-disruption.api.ts

```diff
@@ -0,0 +1,42 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { getPtSituationElementFromDraft } from "./publish.api";
+import { ERROR_PATH } from "../../constants";
+import { deletePublishedDisruption, getDisruptionById } from "../../data/dynamo";
+import { redirectTo, redirectToError } from "../../utils/apiUtils";
+import logger from "../../utils/logger";
+
+const deleteDisruption = async (req: NextApiRequest, res: NextApiResponse): Promise<void> => {
+    try {
+        const { query } = req;
+
+        const id = query?.id as string;
+
+        if (!id) {
```

Comment:
if the user passes through the id as an array (I think thats a potential type it can be?) then we need a check inside this if I think (super niche).

Reply:
done if it is an array an error is thrown